schedule/yast/select_disk/select_disk.yaml

```diff
@@ -39,6 +43,8 @@ conditional_schedule:
       pvm_hmc:
         - boot/reconnect_mgmt_console
 test_data:
-  disks:
-    - vda
+  guided_partitioning:
```

Comment:
If we would like to check the unused disk during installation I would have opted by
  ```yaml
  guided_partitioning:
    disks:
      - name: vda
        used: 1
      - vdb:
        used: 0
But as we don't check everything in the screen, probably is too much,
wdyt?

Reply:
I had this in mind too, but in general I could not come up with any use-case (besides this scenario), where we would have a disk and decide not to use it.